fe/src/components/pages/About/About.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function About(): JSX.Element {
+  // @TODO - Add some actual content here.
+  return (
+    <div>
+      <h1>About Page</h1>
+    </div>
+  );
+}
+
+export default About;
```

Comment:
We should look into [eol-last](https://eslint.org/docs/rules/eol-last)

Reply:
Sure 👍